examples/agent_framework_integrations/google_adk/requirements.txt

```diff
@@ -1,2 +1,4 @@
 google-adk>=1.6.1
 zenml[local]>=0.84.3
+jinja2>=3.1.0
+secure>=0.3.0
```

Comment:
Why were these two added to all the requirements.txt files? I don't see either being used anywhere 🤷

Reply:
hmm id leave it in for safety reasons then